packages/components/src/templates/quickfact.stories.ts

```diff
@@ -16,6 +16,19 @@ export default {
   title: 'Templates/Quickfact'
 };
 
+export const QuickfactWithSummary = {
+  name: 'Quickfact with Summary',
+  render: () =>
+    html` <sd-quickfact class="max-w-[410px]">
```

Comment:
```suggestion
    html` <sd-quickfact class="max-w-[410px] mx-auto">
```

This centers the first quickfact making it consistent with the remaining quickfacts and the figma docs.